requirements.txt

```diff
@@ -1,4 +1,5 @@
-pyquil>=2.16,<2.28.3
+pyquil>=3.0.0,<4.0.0
 pennylane>=0.17
+qcs-api-client>=0.20.13<0.22.0
```

Comment:
What was the need for this?

Reply:
There is [this call](https://github.com/MarquessV/pennylane-forest/pull/1/files#diff-e54edc4229030c98c5b70918872608e43667b52ebfafa4f6c3d93996eea94466R111) to a private method in pyQuil, which requires a `QCSClientConfiguration`. If not for this call, we wouldn't need it.

It gets imported [here](https://github.com/MarquessV/pennylane-forest/pull/1/files#diff-2c834f3933545e6d7c66aa741ee3b0a8f464a08c7d8a0796f202140f54501da1R41) and initialized with [this function](https://github.com/MarquessV/pennylane-forest/pull/1/files#diff-2c834f3933545e6d7c66aa741ee3b0a8f464a08c7d8a0796f202140f54501da1R174-R175)